dist/preview release/what's new.md

```diff
@@ -24,6 +24,7 @@
 - Added a [helper class](https://doc.babylonjs.com/typedoc/classes/babylon.debug.directionallightfrustumviewer) to display the frustum of a directional light ([Popov72](https://github.com/Popov72))
 - Improved collision detection performance ([ottoville](https://github.com/ottoville/))
 - Added new helper functions for Quaternion.FromLookDirection and Matrix.LookDirection ([Alex-MSFT](https://github.com/Alex-MSFT))
+- Improved performance when using the shadow / cascacaded shadow generator ([Popov72](https://github.com/Popov72))
```

Comment:
please move it to main features. This is massive

Reply:
Done!
I also needed to change a bit the UBO binding cache problem because of a circular dependency between `Effect` and `ThinEngine`.